rpc/client/evidence_test.go

```diff
@@ -6,17 +6,20 @@ import (
 	"testing"
 	"time"
 
+	"github.com/line/ostracon/abci/example/kvstore"
```

Comment:
Is IDE or `goimports` the cause? This is strange import order...

Reply:
I think IDE added automatically. I'll re-group and re-sort  under `github.com/line/ostracon/~`